provenance/distroless-provenance-task.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: distroless-provenance
+  annotations:
+    chains.tekton.dev/transparency-upload: "true"
+    description: |
+      Takes in the distroless repo as an input and builds & pushes distroless images.
+spec:
+  results:
+    - name: IMAGES
+  params:
+    - name: MANIFEST_SUBSECTION
+      type: string
+  steps:
+    - name: crane
+      image: golang:1.16
+      env:
+      - name: REGISTRY
+        value: insecure-registry.default.svc.cluster.local:5000
+      script: |
+        #!/bin/sh
+        set -ex
+
+        go install github.com/google/go-containerregistry/cmd/crane@latest
+
+        # install jq and create the images file
+        ./cloudbuild_jq.sh
```

Comment:
Do we need this script still? Same for crane

Reply:
ah yah we don't need crane anymore, removed it

the script basically parses through the manifest and stores all of the images built in a file called `images` which we still use